juniper_official/chassis/check-system-power-usage-temp-state.rule

```diff
@@ -310,6 +264,20 @@ healthbot {
                                     }
                                 }
                             }
+                        }
+                        operating-system junos {							
+                            products MX {
+                                platforms MX204 {
+                                    releases 22.2R3 {
+                                        release-support min-supported-release;
+                                    }
+                                }								
+                                platforms MX304 {
+                                    releases 22.2R3 {
+                                        release-support min-supported-release;
+                                    }
+                                }
+                            }
```

Comment:
Add vmx product as well

Reply:
I have added sensor under Juniper.vmx is not applicable for this rule.